docs/visual-basic/misc/bc30567.md

```diff
@@ -47,5 +47,5 @@ You have not supplied enough elements in the initializer for your array.
 -   Use `ReDim` to change the size of the array.  
   
 ## See Also  
- [NOTINBUILD Overview of Arrays in Visual Basic](http://msdn.microsoft.com/en-us/ca50e2f2-b4d2-4c57-9169-9abbcc3392d8)   
+ [Arrays in Visual Basic](~/docs/visual-basic/programming-guide/language-features/arrays/index.md)   
  [NOTINBUILD Jagged Arrays in Visual Basic](http://msdn.microsoft.com/en-us/05c12439-ee8f-4fef-ba75-b35402b67ab9)
```

Comment:
Yes, I know. I'll let the removal of NIB links for @mikeblome's script. I'm trying to find the replacements before stripping them out.